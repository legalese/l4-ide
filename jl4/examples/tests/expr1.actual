MkProgram
    [ MkSection 1 "Expression priority"
        [ Decide
            ( MkDecide
                ( MkTypeSig [] Nothing )
                [ GuardedClause
                    ( Var "x1" )
                    ( PlainGuard
                        ( And
                            ( Var "foo" )
                            ( Var "bar" )
                        )
                    )
                ]
            )
        , Decide
            ( MkDecide
                ( MkTypeSig [] Nothing )
                [ GuardedClause
                    ( Var "x2" )
                    ( PlainGuard
                        ( And
                            ( Var "foo" )
                            ( Not
                                ( Var "bar" )
                            )
                        )
                    )
                ]
            )
        ]
    ]
