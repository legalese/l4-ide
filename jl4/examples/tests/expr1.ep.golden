ยง `Expression priority`

DECIDE x1 IF     foo
             AND bar

DECIDE x2 IF     foo
             AND NOT bar
